go/store/nbs/table_reader.go

```diff
@@ -596,13 +700,20 @@ func (tr tableReader) extract(ctx context.Context, chunks chan<- extractRecord)
 
 	sendChunk := func(i uint32) error {
 		localOffset := tr.offsets[i] - tr.offsets[0]
-		data, err := tr.parseChunk(buff[localOffset : localOffset+uint64(tr.lengths[i])])
+
+		cmp, err := NewCompressedChunk(hash.Hash(hashes[i]), buff[localOffset:localOffset+uint64(tr.lengths[i])])
```

Comment:
What's the purpose of the change to this function? Seems like equivalent to the old version, but not sure.

Reply:
parseChunk has been removed, and everything uses the CompressedChunk class now so that I don't have to maintain 2 code paths that do the same thing.